protocol-configuration/mainnetAddresses.ts

```diff
@@ -855,6 +823,24 @@ export const MainnetContractsConfig = {
     category: AddressCategory.External
   },
 
+  aaveCompaniesMultisig: {
+    artifactName: 'unknown',
+    address: '0xe8d4D93D9728bD673b0197673a230F62255C7846',
+    category: AddressCategory.External
+  },
+
+  aaveCompaniesDaiEscrowMultisig: {
+    artifactName: 'unknown',
+    address: '0xA27A83A2433EbEB9f658B4420074B454846e1C5B',
+    category: AddressCategory.External
+  },
+
+  escrowedAaveDaiPCVDeposit: {
```

Comment:
please move this to the PCV section above

Reply:
done